[LW-8649] Adds data url for connect source in csp

VanessaPC opened the pull request:
# Checklist

- [x] JIRA - LW-8649
- [ ] Proper tests implemented
- [ ] Screenshots added.

---

## Proposed solution
Adds data URL for connect source in CSP

## Testing
Get the branch, run `yarn cleanup && yarn install` move to `apps/browser...` run `yarn dev` load unpacked extension. It should now work. 

## Screenshots
N/A 


<!-- allure -->
---
# Allure report
`allure-report-publisher` generated test report!

<!-- jobs -->
<!-- smokeTests -->
**smokeTests**: ❌ [test report](https://lace-qa:8PP5wBaDV6UoXj@dq4ajm5i5q7bz.cloudfront.net/linux/chrome/2268/6350907703/index.html) for [0d43fb46](https://github.com/input-output-hk/lace/pull/584/commits/0d43fb46ceb21e42cd7400f26dfe51737fc0ea26)
|       | passed | failed | skipped | flaky | total | result |
|-------|--------|--------|---------|-------|-------|--------|
| Total | 31     | 1      | 0       | 0     | 32    | ❌     |
<!-- smokeTests -->

<!-- jobs -->
<!-- allurestop -->

rhyslbw reviewed (changes requested):
Nice work tracking this down @VanessaPC. It would be good to add some detail to the commit message, as this is lacking context on _why_ the change is needed.

VanessaPC commented:
> Nice work tracking this down @VanessaPC. It would be good to add some detail to the commit message, as this is lacking context on _why_ the change is needed.

Addressed -> 4f2a60ddb9e8966d0df95684502c256ee0554d66  I also updated the ticket for further details.

rhyslbw reviewed (approved):
Nice work @VanessaPC
